keyboards/keebio/iris/keymaps/jasonkrasavage/keymap.c

```diff
@@ -0,0 +1,68 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
```

Comment:
```suggestion
```

Reply:
Thanks, pushed that change. As I mentioned above to @noroadsleft, these changes should be made to the default iris keymap because that is where all of this code came from. For now I have only changed a few key codes. Please let me know if you'd like me to also alter/optimize the default Iris Rev 3 keymap.